data/preferences/src/components/pages/Preferences.vue

```diff
@@ -146,6 +146,23 @@
           ></editable-text-list>
         </td>
       </tr>
+      <tr>
+        <td class="pull-top">
+          <label for="show-all-apps">Show All Apps</label>
+          <small>
+            <p>
+              Ulauncher will display regular applications no matter the value of OnlyShowIn / NotShowIn
+              from the .desktop files.
+            </p>
+            <p v-if="showAllAppsChanged">
+              <i class="fa fa-warning"></i> Restart Ulauncher for this to take effect
+            </p>
+          </small>
+        </td>
+        <td class="pull-top">
+          <b-form-checkbox id="show-all-apps" v-model="show_all_apps"></b-form-checkbox>
```

Comment:
I agree with you about the naming, and do want to get the id right from the start rather than having to change and break people's configs. This is why the review took so long (sorry).

How about `disable-desktop-filters` for the id? That's very technical and direct, which I think is good for the id. The description and name shown for users can still be the same. I prefer it this less technical and without the double negatives I suggested for the id. We can change it at any time, should we want but I think you described it really well :+1: .

Reply:
I've renamed pretty much everything to `disable-desktop-filters`.  Let me know if everything is good.